restli-tools/src/main/java/com/linkedin/restli/tools/data/SchemaFormatTranslator.java

```diff
@@ -268,8 +282,13 @@ private void verifyTranslatedSchemas(Map<String, SchemaInfo> topLevelTranslatedS
         if (!sourceSchema.equals(destSchema))
         {
           LOGGER.error("Translation failed for schema: {}", schemaName);
-          LOGGER.error("Source format: {}, schema:\n{}", _sourceFormat, sourceSchema.toString());
-          LOGGER.error("Destination format: {}, schema:\n{}", _destFormat, destSchema.toString());
+          // Write the translated schema to temp dir.
+          File sourceFile = new File(errorSchemasDir, sourceSchema.getName() + "_" + _sourceFormat);
+          FileUtils.writeStringToFile(sourceFile, SchemaToJsonEncoder.schemaToJson(sourceSchema, JsonBuilder.Pretty.INDENTED));
+          File destFile = new File(errorSchemasDir, sourceSchema.getName() + "_" + _destFormat);
+          FileUtils.writeStringToFile(destFile, SchemaToJsonEncoder.schemaToJson(destSchema, JsonBuilder.Pretty.INDENTED));
+          LOGGER.error("To see the difference between source and tanslated schemas, run: ");
+          LOGGER.error("diff {} {}", sourceFile.getAbsolutePath(), destFile.getAbsolutePath());
```

Comment:
I understand you want to show them in two lines, but following code would feel better

LOGGER.error("To see the difference between source and tanslated schemas, run: \n diff {} {}", sourceFile.getAbsolutePath(), destFile.getAbsolutePath());